content/en/docs/setup/independent/high-availability.md

```diff
@@ -474,69 +340,19 @@ done
     - `ETCD_2_IP`
 
 1.  Run `kubeadm init --config kubeadm-config.yaml`
-1.  Copy the output join commamnd.
-
-### Copy required files to the correct locations
-
-The following pki files were created during the `kubeadm init` step and must be shared with
-all other control plane nodes.
-
-- `/etc/kubernetes/pki/ca.crt`
-- `/etc/kubernetes/pki/ca.key`
-- `/etc/kubernetes/pki/sa.key`
-- `/etc/kubernetes/pki/sa.pub`
-- `/etc/kubernetes/pki/front-proxy-ca.crt`
-- `/etc/kubernetes/pki/front-proxy-ca.key`
-
-In the following example, replace the list of
-`CONTROL_PLANE_IPS` values with the IP addresses of the other control plane nodes.
-
-```sh
-# make a list of required kubernetes certificate files
-cat << EOF > certificate_files.txt
-/etc/kubernetes/pki/ca.crt
-/etc/kubernetes/pki/ca.key
-/etc/kubernetes/pki/sa.key
-/etc/kubernetes/pki/sa.pub
-/etc/kubernetes/pki/front-proxy-ca.crt
-/etc/kubernetes/pki/front-proxy-ca.key
-EOF
-
-# create the archive
-tar -czf control-plane-certificates.tar.gz -T certificate_files.txt
-
-USER=ubuntu # customizable
-CONTROL_PLANE_IPS="10.0.0.7 10.0.0.8"
-for host in ${CONTROL_PLANE_IPS}; do
-    scp control-plane-certificates.tar.gz "${USER}"@$host:
-done
-```
-
-### Set up the other control plane nodes
-
-1.  Extract the required certificates
-
-        mkdir -p /etc/kubernetes/pki
-        tar -xzf etcd-pki.tar.gz -C /etc/kubernetes/pki --strip-components 3
-        tar -xzf control-plane-certificates.tar.gz -C /etc/kubernetes/pki --strip-components 3
-
-1.  Verify the location of the copied files.
-    Your `/etc/kubernetes` directory should look like this:
+1.  Write the join command to a text file for later use.
```

Comment:
It should be useful to give an example of kubectl get pods highlighting the etcd pods missing versus the same operation above